### ISSUE TITLE
drush-install runs every time puppet runs

### ISSUE THREAD

**Issue by tdm4** [previousnext/puppet-drush#3]
Every time puppet runs, the drush-install component runs:

Info: Applying configuration version '1455273389'
Notice: /Stage[main]/Drush/Drush::Drush[drush8]/Exec[drush-install-8]/returns: executed successfully

Hiera:
```
classes:
  - composer
  - drush
drush:
  drush8:
    version:   '8'
    link_name: '/usr/local/bin/drush'
```
Should there be an onlyif/unless test in exec { "drush-install-${version}"?

**Comment by tdm4** [comment]
Would this work?
```
  # Grab drush via composer.
  exec { "drush-install-${version}":
    command     => "composer --prefer-dist require drush/drush ${version}",
    path        => '/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin',
    cwd         => "${target_dir}/drush-${version}",
    environment => "COMPOSER_HOME=${target_dir}/drush-${version}",
    onlyif      => "test ! -f ${target_dir}/drush-${version}/vendor/bin/drush",
    user        => $user,
    require     => [
      Class['composer']
    ]
  }
```

**Comment by atrepca** [comment]
Bumped into the same issue. @cafuego, need a PR for fixing this?

**Comment by cafuego** [comment]
Thanks for the PR! Sorry, I totally missed the notification for this. Looks good, so merging. I think a possible improvement in the future would be to check that the version numbers match.

**Comment by cafuego** [comment]
Fixed in 0.1.2. Thanks again!
